Files/Views/LayoutModes/GenericFileBrowser.xaml.cs

```diff
@@ -543,7 +543,14 @@ private void AllView_PreviewKeyDown(object sender, KeyRoutedEventArgs e)
                 }
                 else
                 {
-                    NavigationHelpers.OpenSelectedItems(ParentShellPageInstance, false);
+                    if (AppSettings.OpenFoldersNewTab)
```

Comment:
This feature still only works in the details view browser. It might be easier to move this check to ```NavigationHelpers.OpenSelectedItems``` so that it is opened in a new tab regardless of the layout.